package.json

```diff
@@ -111,6 +111,6 @@
     "tailwindcss": "3.2.4",
     "ts-loader": "8.4.0",
     "typescript": "4.9.4",
-    "webpack": "5.76.1"
+    "webpack": "5.74.0"
```

Comment:
Can we keep `webpack` at version `5.76.1` please?

Reply:
for sure. just in a mode of throwing stuff at the wall to see what sticks. after the cache clear build, I'll get package.json up-to-date with `main`.